e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -109,19 +130,23 @@ describe('team management', () => {
     const dropdownNameUntilEllipsisLen = 25;
 
     context('when only the unassigned project is selected', () => {
-      before(() => {
+      beforeEach(() => {
         localStorage.setItem(STORE_OPTIONS_KEY, JSON.stringify([
-          {label: unassigned, value: unassigned, checked: true},
-          {label: project1Name, value: project1ID, checked: false},
-          {label: project2Name, value: project2ID, checked: false},
-          {label: unusedProject, value: unusedProject, checked: false}
+          {label: unassigned, value: unassigned, type: 'CUSTOM', checked: true},
+          {label: project1Name, value: project1ID, type: 'CUSTOM', checked: false},
+          {label: project2Name, value: project2ID, type: 'CUSTOM', checked: false},
+          {label: unusedProject, value: unusedProject, type: 'CUSTOM', checked: false}
         ]));
-
-        // relog to apply localstorage state
         cy.adminLogin('/settings/teams').then(() => {
           const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
           adminToken = admin.id_token;
+          cy.cleanupTeamsByDescriptionPrefix(adminToken, cypressPrefix);
+          cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
         });
+        cy.restoreStorage();
+        // Necessary to pick up localstorage changes
```

Comment:
Another hack we wouldn't need if we selected projects by clicking 😉

Reply:
I'm gonna merge but doing a follow up PR immediately after! I'll roll these into that PR.